packages/rulesets/src/spectral/functions/avoid-additional-properties.ts

```diff
@@ -0,0 +1,18 @@
+const avoidAdditionalProperties = (schema: any, _opts: any, paths: any) => {
+  if (schema === null) {
+    return []
+  }
+  const path = paths.path || {}
+  if ("tags" === path[path.length - 1] && "additionalProperties" in schema) {
```

Comment:
How is this logic addressing the issue with common types having additionalproperties?

Reply:
Logic is same, I will remove this file. I just modified the resolved to false.